app/assets/stylesheets/repository/index.scss

```diff
@@ -11,6 +11,12 @@
     }
   }
 
+  .toolbar {
+    .hidden {
+      display: none;
+    }
+  }
+
```

Comment:
Isn't there a similar style already defined? I would imagine it should already be somewhere, maybe even as a general offering from bootstrap.

Reply:
Yes it is bootstrap global class